compiler/integration-tests/test/integration/browser/compile_prove_verify.test.ts

```diff
@@ -1,22 +1,20 @@
-import { expect } from '@esm-bundle/chai';
+import { expect } from "@esm-bundle/chai";
 import { initializeResolver } from "@noir-lang/source-resolver";
 import newCompiler, {
-    compile,
-    init_log_level as compilerLogLevel
+  compile,
+  init_log_level as compilerLogLevel,
 } from "@noir-lang/noir_wasm";
-import { decompressSync as gunzip } from 'fflate';
+import { decompressSync as gunzip } from "fflate";
 import newABICoder, { abiEncode } from "@noir-lang/noirc_abi";
 import initACVM, {
-    executeCircuit,
-    WitnessMap,
-    compressWitness,
+  executeCircuit,
+  WitnessMap,
+  compressWitness,
 } from "@noir-lang/acvm_js";
 
-// @ts-ignore
```

Comment:
Is the ts-ignore not needed anymore?

Reply:
We only need this because @aztec/bb.js doesn't provide any types. It's a cleaner solution to just provide a declarations file than using `ts-ignore`.

https://medium.com/@steveruiz/using-a-javascript-library-without-type-declarations-in-a-typescript-project-3643490015f3